Modules/xASL_module_ASL.m

```diff
@@ -189,6 +190,34 @@
     x.modules.asl.bMultiTE = false;
 	x.Q.NumberEchoTimes = 1;
 end
+%% D6. Automatic session merging
```

Comment:
Why don't we also do this in the separate merging function? This is only needed there, so it makes sense to only do it there. Plus, it here makes xASL_module_ASL busier for an exotic option.

Reply:
It's just checking params and making sure that the parsing list is OK. It's not doing any merging really.

Previously, you wanted to have all the general parameter checking at the start of ASL module...

Reply:
Again - you very specifically asked 1 year ago to move bMultiTE here and you strongly argued that you absolutely want to have all these parameters here and that we absolutely shouldn't have any parameters like bMultiTE anywhere else but at the start of ASL-module.

You have asked for bMultiTE and bHadamard and all these parameters to be specifically here. That is your decision from 1 year ago and you insisted on this.

It is not only used in merging function, but also in motion correction function, in quantification function and will be possibly also used elsewhere.

Reply:
Yes for bMultiTE and bHadamard, but no for everything that is used on a single location only. Why do we use bSessionMerging in motion correction? I would say that we either merge BEFORE motion correction etc or AFTER but the motion correction etc can be oblivious to session merging. Doing session merging at multiple locations makes for parallel pipeline routes which complicates debugging etc.